python/cudf/io/csv.py

```diff
@@ -42,6 +43,14 @@ def read_csv(filepath, lineterminator='\n',
     dtype : list of str or dict of {col: dtype}, default None
         List of data types in the same order of the column names
         or a dictionary with column_name:dtype (pandas style).
+    quotechar : char, default '\0' (empty)
```

Comment:
Per Pandas's API, shouldn't the quotechar default be '"' instead of empty?

Reply:
You're right! I was only looking at the param description in the docs, where it didn't specify the default. Should've looked at the function declaration too. Fixed.